### DISCUSSION TITLE
Im Disabledandwon'ttobeavolunteer<title>

### DISCUSSION THREAD

**Discussion by Disabled83** [community/community#48659] [Accessibility]
### Select Topic Area

Question

### Body

Hi there I'm Disabled and would like to volunteer my time to help where it is needed. I don't have a computer just a phone. If I could get a chance to help out it would be awesome. If there is someone that would like to mentor me that would be awesome too. I'm very smart and a quick learner I just have to much time on my hands and would like to put them to use somewhere you can contact me at disabledlife87@gmail.com

**Comment by cehfisher** [comment]
Hi @Disabled83 - are you still interested in helping out with accessibility testing? If so, please open a ticket on our [Support page](https://support.github.com/contact) and we will follow-up with next steps.
